src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -1304,6 +1370,13 @@ fixCopyCommand(Archive *fout, PQExpBuffer copyBuf, TableInfo *tbinfo, bool isFro
 		 */
 		else if (is_bbf_sysdatabases_tab && strcmp(tbinfo->attnames[i], "owner") == 0)
 			continue;
+		/*
+		 * Skip scheme_id column of babelfish_partition_scheme catalog
```

Comment:
We always dump data with insert for tables having sql_variant columns. Since `babelfish_partition_function` catalog contains sql_variant, we don't need to handle dump with copy for this catalog.